Pandoc causes build failure

Latest github build:

```
  py38-jinja2211 run-test-pre: PYTHONHASHSEED='3003592029'
  py38-jinja2211 run-test: commands[0] | coverage run -a --branch -m unittest
  ................................/Users/runner/hostedtoolcache/Python/3.8.6/x64/lib/python3.8/tempfile.py:818: ResourceWarning: Implicitly cleaning up <TemporaryDirectory '/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmp88hgo7dx'>
    _warnings.warn(warn_message, ResourceWarning)
  /Users/runner/hostedtoolcache/Python/3.8.6/x64/lib/python3.8/tempfile.py:818: ResourceWarning: Implicitly cleaning up <TemporaryDirectory '/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmpv1w_dshr'>
    _warnings.warn(warn_message, ResourceWarning)
  ./Users/runner/hostedtoolcache/Python/3.8.6/x64/lib/python3.8/tempfile.py:818: ResourceWarning: Implicitly cleaning up <TemporaryDirectory '/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmp2_orl_0t'>
    _warnings.warn(warn_message, ResourceWarning)
  ................F.........................................................127.0.0.1 - - [05/Dec/2020 07:07:42] code 404, message File not found
  127.0.0.1 - - [05/Dec/2020 07:07:42] "GET /not-existing.html HTTP/1.1" 404 -
  .127.0.0.1 - - [05/Dec/2020 07:07:42] "GET /style HTTP/1.1" 200 -
  127.0.0.1 - - [05/Dec/2020 07:07:42] code 404, message File not found
  127.0.0.1 - - [05/Dec/2020 07:07:42] "GET /page HTTP/1.1" 404 -
  .127.0.0.1 - - [05/Dec/2020 07:07:43] "GET /blog/ HTTP/1.1" 200 -
  .127.0.0.1 - - [05/Dec/2020 07:07:43] "GET /blog/ HTTP/1.1" 200 -
  .127.0.0.1 - - [05/Dec/2020 07:07:44] "GET /style.css HTTP/1.1" 200 -
  .127.0.0.1 - - [05/Dec/2020 07:07:44] "GET /style.css#something HTTP/1.1" 200 -
  .127.0.0.1 - - [05/Dec/2020 07:07:45] "GET /style.css?something=1 HTTP/1.1" 200 -
  .127.0.0.1 - - [05/Dec/2020 07:07:45] "GET /bob/style.css HTTP/1.1" 200 -
  127.0.0.1 - - [05/Dec/2020 07:07:45] code 404, message File not found
  127.0.0.1 - - [05/Dec/2020 07:07:45] "GET /style.css HTTP/1.1" 404 -
  .127.0.0.1 - - [05/Dec/2020 07:07:46] "GET / HTTP/1.1" 200 -
  .127.0.0.1 - - [05/Dec/2020 07:07:46] "GET /page HTTP/1.1" 200 -
  .127.0.0.1 - - [05/Dec/2020 07:07:47] "GET /page/ HTTP/1.1" 200 -
  .127.0.0.1 - - [05/Dec/2020 07:07:48] code 404, message File not found
  127.0.0.1 - - [05/Dec/2020 07:07:48] "GET /unknown HTTP/1.1" 404 -
  .127.0.0.1 - - [05/Dec/2020 07:07:48] "GET /page.html HTTP/1.1" 200 -
  .
  ======================================================================
  FAIL: test_pandoc_filter_no_argument (exhibition.tests.test_filters.Jinja2TestCase)
  ----------------------------------------------------------------------
  Traceback (most recent call last):
    File "/Users/runner/work/exhibition/exhibition/exhibition/tests/test_filters.py", line 288, in test_pandoc_filter_no_argument
      self.assertEqual(result, "\nHello\n=====\n\nThis *is* **text**\n")
  AssertionError: '\n# Hello\n\nThis *is* **text**\n' != '\nHello\n=====\n\nThis *is* **text**\n'
    
  - # Hello
  ? --
  + Hello
  + =====
    
    This *is* **text**
  
  
  ----------------------------------------------------------------------
  Ran 120 tests in 8.828s
  
  FAILED (failures=1)
  ERROR: InvocationError for command /Users/runner/work/exhibition/exhibition/.tox/py38-jinja2211/bin/coverage run -a --branch -m unittest (exited with code 1)
```

--- a/exhibition/tests/test_filters.py
+++ b/exhibition/tests/test_filters.py
@@ -271,6 +271,10 @@ class Jinja2TestCase(TestCase):
         )
 
     def test_pandoc_filter_no_argument(self):
+        possible_markdown_outputs = [
+            "\nHello\n=====\n\nThis *is* **text**\n",
+            "\n# Hello\n\nThis *is* **text**\n",
+        ]
         meta = {"templates": []}
         node = Node(mock.Mock(), None, meta=meta)
         node.is_leaf = False
@@ -285,7 +289,7 @@ class Jinja2TestCase(TestCase):
 
         node.meta["pandoc_config"] = {"format": "org", "to": "markdown"}
         result = jinja_filter(node, PANDOC_TEMPLATE_WITHOUT_ARG)
-        self.assertEqual(result, "\nHello\n=====\n\nThis *is* **text**\n")
+        self.assertIn(result, possible_markdown_outputs)
 
     def test_typogrify_filter(self):
         node = Node(mock.Mock(), None, meta={"templates": []})

